### PR TITLE
Persistence 3.0 and Concurrency 2.0 TCK docs

### PR DESCRIPTION
**Pull request by edbratt**
Persistence 3.0 and Concurrency Utilities 2.0 TCK Documents revision -- a couple of outstanding issues remain (though this can be merged):
1) Need someone to review the tested products matrix in the release notes (have asked @lukasj for help)
2) Need to confirm Apache Ant version, if included.

### CODE REVIEW

File: user_guides/jpa/src/main/jbake/templates/footer.ftl [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
 <span id="copyright">
 		<img src="img/eclipse_foundation_logo_tiny.png" height="20px" alt="Eclipse Foundation Logo" align="top"/>&nbsp;			
-		<span >Copyright&nbsp;&copy;&nbsp;2018,&nbsp;Oracle&nbsp;and/or&nbsp;its&nbsp;affiliates.&nbsp;All&nbsp;rights&nbsp;reserved.</span>
+		<span >Copyright&nbsp;&copy;&nbsp;2018,&nbsp2020&nbsp;Oracle&nbsp;and/or&nbsp;its&nbsp;affiliates.&nbsp;All&nbsp;rights&nbsp;reserved.</span>
```
Comment (alwin-joseph) [inline user_guides/jpa/src/main/jbake/templates/footer.ftl:40]: One ; missing before 2020

File: user_guides/concurrency/src/main/jbake/content/tck-packages.inc [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
 ** Apache Commons HTTP Client 3.1
 
-** Apache Commons Logging 1.1.1
+** Apache Commons Logging 1.1.3
 
-** Apache Commons Codec 1.3
+** Apache Commons Codec 1.9
 
 ** Apache Ant 1.8.4
```
Comment (alwin-joseph) [inline user_guides/concurrency/src/main/jbake/content/tck-packages.inc:9]: Ant version 1.10.8

File: user_guides/concurrency/src/main/jbake/templates/footer.ftl [hunk 3]
```diff
@@ -37,7 +37,7 @@
 
 <span id="copyright">
 		<img src="img/eclipse_foundation_logo_tiny.png" height="20px" alt="Eclipse Foundation Logo" align="top"/>&nbsp;			
-		<span >Copyright&nbsp;&copy;&nbsp;2018,&nbsp;Oracle&nbsp;and/or&nbsp;its&nbsp;affiliates.&nbsp;All&nbsp;rights&nbsp;reserved.</span>
+		<span >Copyright&nbsp;&copy;&nbsp;2018,&nbsp2020&nbsp;Oracle&nbsp;and/or&nbsp;its&nbsp;affiliates.&nbsp;All&nbsp;rights&nbsp;reserved.</span>
```
Comment (alwin-joseph) [inline user_guides/concurrency/src/main/jbake/templates/footer.ftl:40]: One ; missing before 2020

File: install/concurrency/docs/CONCURRENCY2.0-ReleaseNotes.html [hunk 4]
```diff
@@ -77,22 +77,21 @@ <h2><a name="platform">Platform Notes</a></h2>
 <p>The Jakarta Concurrency TCK tests have been run on the following platforms:</p>
 <ul>
 <li>MAC OS X, <em>El Capitan</em> (10.9.5+)</li>
-<li>Oracle Linux 7.1</li>
-<li>Windows 10</li>
+<li>CentOS Linux 7</li>
 </ul>
```
Comment (scottmarlow) [inline install/concurrency/docs/CONCURRENCY2.0-ReleaseNotes.html:81]: :+1:

File: install/concurrency/docs/CONCURRENCY2.0-ReleaseNotes.html [hunk 5]
```diff
@@ -77,22 +77,21 @@ <h2><a name="platform">Platform Notes</a></h2>
 <p>The Jakarta Concurrency TCK tests have been run on the following platforms:</p>
 <ul>
 <li>MAC OS X, <em>El Capitan</em> (10.9.5+)</li>
```
Comment (scottmarlow) [inline install/concurrency/docs/CONCURRENCY2.0-ReleaseNotes.html:79]: I would remove ```MAC OS X``` unless we have heard from someone that was actually used to pass the Jakarta Concurrency 2.0 TCK tests with.

File: user_guides/concurrency/src/main/jbake/templates/footer.ftl (hunk 3 above)
Comment (edbratt) [reply user_guides/concurrency/src/main/jbake/templates/footer.ftl]: Done

File: user_guides/concurrency/src/main/jbake/content/tck-packages.inc (hunk 2 above)
Comment (edbratt) [reply user_guides/concurrency/src/main/jbake/content/tck-packages.inc]: Thank you, updated

File: user_guides/jpa/src/main/jbake/templates/footer.ftl (hunk 1 above)
Comment (edbratt) [reply user_guides/jpa/src/main/jbake/templates/footer.ftl]: Fixed

File: install/concurrency/docs/CONCURRENCY2.0-ReleaseNotes.html (hunk 5 above)
Comment (edbratt) [reply install/concurrency/docs/CONCURRENCY2.0-ReleaseNotes.html]: I pulled it out. If lead wishes, we can change it back

**Pull request merged**